styles.css

```diff
@@ -9,6 +9,7 @@ body{
 }
 header, footer, aside, .main{
   padding: 1em;
+  margin: 0;
```

Comment:
This will set `margin: 0` for tags themselves, but we wanted to target all of their children. We can do that with child selector `>`

```css
header > *, footer > *, .main > *, aside > * {
   margin: 0;
}
```